### PR TITLE
docs(select): size demo change hook

### PR DESCRIPTION
**Pull request by hengkx**
[[中文版模板 / Chinese template](https://github.com/ant-design/ant-design/blob/master/.github/PULL_REQUEST_TEMPLATE/pr_cn.md)]

### 🤔 This is a ...

- [ ] New feature
- [ ] Bug fix
- [x] Site / documentation update
- [ ] Demo update
- [ ] Component style update
- [ ] TypeScript definition update
- [ ] Bundle size optimization
- [ ] Performance optimization
- [ ] Enhancement feature
- [ ] Internationalization
- [ ] Refactoring
- [ ] Code style optimization
- [ ] Test Case
- [ ] Branch merge
- [ ] Other (about what?)

### 📝 Changelog

| Language   | Changelog |
| ---------- | --------- |
| 🇺🇸 English |           |
| 🇨🇳 Chinese |           |

### ☑️ Self Check before Merge

⚠️ Please check all items below before review. ⚠️

- [ ] Doc is updated/provided or not needed
- [ ] Demo is updated/provided or not needed
- [ ] TypeScript definition is updated/provided or not needed
- [ ] Changelog is provided or not needed

-----
[View rendered components/select/demo/size.md](https://github.com/ant-design/ant-design/blob/im/dc/components/select/demo/size.md)

### CODE REVIEW

File: components/select/demo/size.md [hunk 1]
```diff
@@ -27,55 +27,50 @@ function handleChange(value) {
...
+  return (
+    <>
+      <Radio.Group value={size} onChange={handleSizeChange}>
+        <Radio.Button value="large">Large</Radio.Button>
+        <Radio.Button value="default">Default</Radio.Button>
+        <Radio.Button value="small">Small</Radio.Button>
+      </Radio.Group>
+      <br />
+      <br />
+      <Select size={size} defaultValue="a1" onChange={handleChange} style={{ width: 200 }}>
```
Comment (xrkffgg) [inline components/select/demo/size.md:46]: handleChange 这个没定义？

Comment (hengkx) [reply components/select/demo/size.md:46]: 定义了 在上边

Comment (xrkffgg) [reply components/select/demo/size.md:46]: 噢噢，看到了 😂

**Review by xrkffgg — APPROVED** [review]

File: components/select/demo/size.md (hunk 1 above)
Comment (hengkx) [reply components/select/demo/size.md:46]: 你一说 吧我 吓一跳，直接看不到😅

**Pull request merged**